Sources/SwifQL/SwifQLSelectBuilder.swift

```diff
@@ -8,18 +8,33 @@
 import Foundation
 
 public class SwifQLSelectBuilder {
-    var select: [SwifQLable] = []
-    var froms: [SwifQLable] = []
-    var joins: [SwifQLJoinBuilder] = []
-    var wheres: [SwifQLable] = []
-    var groupBy: [SwifQLable] = []
-    var havings: [SwifQLable] = []
-    var orderBy: [OrderByItem] = []
-    var offset: Int?
-    var limit: Int?
+    private var select: [SwifQLable] = []
+    private var froms: [SwifQLable] = []
+    private var joins: [SwifQLJoinBuilder] = []
+    private var wheres: [SwifQLable] = []
+    private var groupBy: [SwifQLable] = []
+    private var havings: [SwifQLable] = []
+    private var orderBy: [OrderByItem] = []
+    private var offset: Int?
+    private var limit: Int?
     
     public init() {}
     
+    public func copy() -> SwifQLSelectBuilder {
+        let copy = SwifQLSelectBuilder()
+        
+        copy.select = self.select
```

Comment:
Could you change (lines 26-34) `copy.XXX = self.XXX` to `copy.XXX = XXX` - to stay in my code style (don't use self until it needed for compiler)

Reply:
done